src/Services/LoginService.js

```diff
@@ -0,0 +1,40 @@
+import { userLoginValidationSchema } from "../validationSchemas/userValidationSchemas.js";
+import bcrypt from 'bcrypt';
+
+const salt = 10;
+var users  = [
+    {
+        username: "TestUser",
+        password: "$2b$10$FC8ScEhPLdNsWTDzeZ/gxerqW53SMLMsJ76C.irBdw.U1mfjBBys6",
+        email: null
+    }
+];
+
+const login = async (req, res) => {
+    const { error, value } = userLoginValidationSchema.validate(req.body);
+
+    if (error) {
+        return {
+            'code': 401,
+            'body':  error.details[0].message
+        }
+      } else {
+        for (const user of users) {
+            const passwordVerify = await bcrypt.compare(value.password, user.password);
```

Comment:
bcrypt.compare - hard operation, for what do you call this function for each user in users array?
image that you have 1m users how much time you will spend if you compare passwords each of them?
you should 
find user with the passed name in users array.
If user exists, check password using bcrypt.compare.